app/models/vim_performance_daily.rb

```diff
@@ -25,7 +25,7 @@ def self.excluded_cols_for_expressions
   def self.find(cnt, *args)
     raise "Unsupported finder value #{cnt}" unless cnt == :all
 
-    Vmdb::Deprecation.deprecation_warning(:find, :find_entries)
+    Vmdb::Deprecation.deprecation_warning(:find, :find_entries, caller(1))
```

Comment:
I'm afraid not, maybe if we're extending it, it might add a single location to caller that rails can't predict, I'll double check.